MultiplayerMod/Game/Mechanics/SliderEvents.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using HarmonyLib;
+using MultiplayerMod.Core.Patch;
+using MultiplayerMod.Multiplayer.Objects;
+
+namespace MultiplayerMod.Game.Mechanics;
+
+public static class SliderEvents {
+
+    public static event Action<MultiplayerReference, float, int> SetSliderValue;
+
+    [HarmonyPatch]
```

Comment:
You can use it on outer class without any problems.

Reply:
Done